module-check/src/test/clojure/clojure/core/typed/test/template.clj

```diff
@@ -0,0 +1,8 @@
+(ns test.template
+(:require [clojure.core.typed :as t]
+          [clojure.template :as template]
+                [test.core]))
+
+
+(template/apply-template '[a b c d e] '[d a b e c e b a d] '(1 2 3 4 5))
```

Comment:
Can you use `is-tc-e` and `is-tc-err` functions to test specific annotations? Currently you test only correct usages, but it would be good to test incorrect usages so that annotations actually catch errors. See examples here: https://github.com/clojure/core.typed/pull/11/files#diff-97b4dfca878dc0ed8badb483dc921e72R1105

This way you can keep all tests in `test/template.clj` and you don't need to create `core_test.clj` at all.